### PR TITLE
fix_bugs:add initialStyleClass

### PR DESCRIPTION
**Pull request by behroozbc**
###Defect Fixes
https://github.com/primefaces/primeng/issues/8996
hi 
my vscode style is deferent

### CODE REVIEW

**Comment by alixdehghani** [comment]
this issue is fix in pull request https://github.com/primefaces/primeng/pull/9020

**Comment by robybc** [comment]
hi @yigitfindikli
can you  review

**Comment by MaKCbIMKo** [comment]
I'm not the person to review PRs here, but you have changed file formatting -> it makes it really hard to see only required changes + it differs from the whole code base (so, it might not be merged)

**Comment by robybc** [comment]
@MaKCbIMKo hi 
thanks 
I fixed

**Comment by yigitfindikli** [comment]
Hi @behroozbc, thanks for the contribution.

File: src/app/components/button/button.ts [hunk 1]
```diff
@@ -66,7 +69,7 @@ export class ButtonDirective implements AfterViewInit, OnDestroy {
 
     setStyleClass() {
         let styleClass = this.getStyleClass();
-        this.el.nativeElement.className = styleClass;
+        this.el.nativeElement.className = styleClass + this._initialStyleClass;
```
Comment (jmesa-sistel) [inline src/app/components/button/button.ts:72]: It should be:
`this.el.nativeElement.className` = styleClass + ' ' + this._initialStyleClass;

Comment (behroozbc) [reply src/app/components/button/button.ts:72]: yes i fix this in https://github.com/primefaces/primeng/pull/9041

**Pull request merged**